src/app/admin/auditLogTable/auditLogModal/AuditLogModal.tsx

```diff
@@ -0,0 +1,113 @@
+    setModalIsOpen,
+    selectedAuditLogRow,
+}) => {
+    const theme = useTheme();
+
+    return (
+        <Modal
+            header={
+                <>
+                    <Icon icon={auditLogIcon} color={theme.primary.largeForeground[2]} />
+                    {selectedAuditLogRow?.action
+                        ? capitaliseWords(selectedAuditLogRow?.action)
+                        : ""}
+                </>
+            }
+            isOpen={modalIsOpen}
+            onClose={() => {
+                setModalIsOpen(false);
+            }}
+            headerId="auditLogModal"
+        >
+            {selectedAuditLogRow && (
+                <AuditLogModalContainer>
+                    {selectedAuditLogRow.parcelId && (
+                        <ParcelAuditLogModalRow parcelId={selectedAuditLogRow.parcelId} />
+                    )}
+                    {selectedAuditLogRow.clientId && (
+                        <ClientAuditLogModalRow clientId={selectedAuditLogRow.clientId} />
+                    )}
+                    {selectedAuditLogRow.collectionCentreId && (
+                        <CollectionCentreAuditLogModalRow
+                            collectionCentreId={selectedAuditLogRow.collectionCentreId}
+                        />
+                    )}
+                    {selectedAuditLogRow.eventId && (
+                        <EventAuditLogModalRow eventId={selectedAuditLogRow.eventId} />
+                    )}
+                    {selectedAuditLogRow.profileId && (
+                        <ProfileAuditLogModalRow profileId={selectedAuditLogRow.profileId} />
+                    )}
+                    {selectedAuditLogRow.listId && (
+                        <ListsAuditLogModalRow listsId={selectedAuditLogRow.listId} />
+                    )}
+                    {selectedAuditLogRow.listHotelId && (
+                        <ListsHotelAuditLogModalRow
+                            listsHotelId={selectedAuditLogRow.listHotelId}
+                        />
+                    )}
+                    {selectedAuditLogRow.packingSlotId && (
+                        <PackingSlotAuditLogModalRow
+                            packingSlotId={selectedAuditLogRow.packingSlotId}
+                        />
+                    )}
+                    {selectedAuditLogRow.statusOrder && (
+                        <StatusOrderAuditLogModalRow
+                            statusOrderEventName={selectedAuditLogRow.statusOrder}
+                        />
+                    )}
+                    {selectedAuditLogRow.websiteData && (
+                        <WebsiteDataAuditLogModalRow
+                            websiteDataName={selectedAuditLogRow.websiteData}
+                        />
+                    )}
+                    <AuditLogModalItem>
+                        <Key>ACTION CONTENT: </Key>
+                        <TextValueContainer>
+                            {formatJson(selectedAuditLogRow.content)}
```

Comment:
could we change formatJson to use something like `JSON.stringify(json, null, 2)` to see if that makes it prettier?
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/JSON/stringify

Reply:
Yep it's slightly better